packages/web/scripts/security.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+tempFile=$(mktemp)
```

Comment:
We shouldn't create new files outside of `/etc/nginx` - the rest of filesystem may be read-only